Copying.ipynb

```diff
@@ -0,0 +1,1063 @@
+{
```

Comment:
<p>Add heading here like "additional examples" or "Lets play with it some more" or so to mark this part somehow? Now its just a bunch of random lines pretty much ( I actually think they can just be removed?!)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuefische/ds-meetups/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
ups... haven't removed them :D